osbenchmark/workload_generator/helpers.py

```diff
@@ -75,6 +75,15 @@ def make_workload_directory(self):
         io.ensure_dir(self.custom_workload.operations_path)
         io.ensure_dir(self.custom_workload.test_procedures_path)
 
+    def write_custom_workload_record(self, template_vars):
```

Comment:
Where does this generated file get used?

Reply:
I've seen two reoccurring use-cases with create-workload:
- Users using it iteratively to produce the ideal workload. This might include creating several variations of workloads of different sizes and different indices.
- Users using create-workload in different screens to extract documents in parallel. This is especially helpful when there are large indices that must be extracted. 

This generated file serves as historic record in case users modify the workload.json and can also be a way for a future feature in OSB to "merge" workloads together. This is a tactic that I've also introduced into the Synthetic Data Generator I'm implementing.

Reply:
I think this is a good idea.  Should be properly documented if it isn't already.